recipes/entityx/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.33.0"
```

Comment:
```suggestion
required_conan_version = ">=1.32.0"
```
validate() is 1.32.0